data_acquisition/search_generic.py

```diff
@@ -31,7 +31,7 @@
 from tweepy.parsers import RawParser
 import sys
 import argparse
-from auth import TwitterAuth
+from copy_auth import TwitterAuth
```

Comment:
you dont need copy_auth; you need to rename copy_auth to auth.py. copy_auth is just a format of how auth.py should look like

Reply:
I removed remote from the upstream, how is it still committing to this repository?

Do not, merge this, its all experimental and should be updated in my fork.

Reply:
Yea. I got a pull request and @jmbanda  assigned me to review. You are committing to the master and not your fork/master

Reply:
Okay, I understand, this was an early pull req i made with CSV writing issues. I've been committing to my branch and that might have brought this back up. I will close the pull req.